hospital webapplication/src/com/itt/hospital/model/EmployeeData.java

```diff
@@ -0,0 +1,54 @@
+package com.itt.hospital.model;
+
+import java.util.Iterator;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+
+import com.itt.hospital.pojo.Employee;
+
+public class EmployeeData {
+
+	StandardServiceRegistry ssr = new StandardServiceRegistryBuilder().configure("hibernate.cfg.xml").build();
```

Comment:
Creating Session factory you are doing in each class can we move this common code on single place.